pt.html

```diff
@@ -161,36 +161,42 @@
 <panel w=500 h=250>
 <pic src="pics/intro0.png" sx=400 sy=250></pic>
 <words x=155 y=20 w=90 no-bg>
-	Yeah.
+	É assim.
 </words>
 <words x=196 y=65 w=270 no-bg>
-	Not only are common practices like lectures, cramming, and re-reading <i>boring</i>,
-	science has shown they <i>don't even work well</i>.*
+	Não apenas as práticas comuns, como palestras, <i><abbr title="Ato de decorar quantidades massivas no dia anterior de um teste" lang="en">cramming</abbr></i>
+	e releituras são <i>chatas</i>,
+	a ciência tem mostrado que <i>nem mesmo funcionam bem</i>.*
```

Comment:
```suggestion
	a ciência mostra que <i>nem funcionam bem</i>.*
```
Tentando tirar alguns caracteres para ficar dentro do balão

Reply:
```suggestion
	a ciência tem mostrado que <i>nem sequer funcionam bem</i>.*
```